cli/dcoscli/data/config-schema/cosmos.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "url": {
+      "type": "string",
+      "format": "uri",
+      "title": "Cosmos base URL",
+      "description": "Base URL for talking to COSMOS. It overwrites the value specified in core.dcos_url",
+      "default": "http://localhost:7070"
+    }
+  },
+  "additionalProperties": false
+}
```

Comment:
We may want to change additionalProperties to true after your testing.
Thanks!

On Tue, Feb 9, 2016 at 4:15 PM tamarrow <notifications@github.com> wrote:

> In cli/dcoscli/data/config-schema/cosmos.json
> <https://github.com/mesosphere/dcos-cli/pull/456#discussion_r52397612>:
>
> > @@ -0,0 +1,14 @@
> > +{
> > +  "$schema": "http://json-schema.org/schema#",
> > +  "type": "object",
> > +  "properties": {
> > +    "url": {
> > +      "type": "string",
> > +      "format": "uri",
> > +      "title": "Cosmos base URL",
> > +      "description": "Base URL for talking to COSMOS. It overwrites the value specified in core.dcos_url",
> > +      "default": "http://localhost:7070"
> > +    }
> > +  },
> > +  "additionalProperties": false
> > +}
>
> ok. I'm going to remove package.sources and package.cache in a following
> commit because it makes this a much larger change.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/dcos-cli/pull/456/files#r52397612>.
>
-- 
José Armando García Sancio
Mesosphere